cmd/ls_test.go

```diff
@@ -0,0 +1,43 @@
+package cmd
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLsNoErr(t *testing.T) {
```

Comment:
Maybe this should not have the error part?

```suggestion
func TestLs(t *testing.T) {
```